dhall/src/Dhall/Parser.hs

```diff
@@ -82,7 +84,7 @@ exprFromText
             --   used in parsing error messages
   -> Text   -- ^ Input expression to parse
   -> Either ParseError (Expr Src Import)
-exprFromText delta text = fmap snd (exprAndHeaderFromText delta text)
+exprFromText delta text = fmap snd (exprAndHeaderFromText CommentIsWhitespace delta text)
```

Comment:
Todo: take this as an argument just like in `exprAndHeaderFromText`. Caller's responsibility to set comment control.

Reply:
Do you still plan to do this?

Reply:
I am actually not sure any more. This `exprFromText` function seems to be used in many places. It would be convenient to not have to specify the `CommentControl`/`WhitespaceControl` when using it. I don't think this control is really necessary outside of `format`, `freeze`, and `lint` anyway.